lighthouse-core/gather/gatherers/seo/font-size.js

```diff
@@ -294,11 +299,17 @@ class FontSize extends Gatherer {
       .sort((a, b) => b.textLength - a.textLength)
       .slice(0, MAX_NODES_SOURCE_RULE_FETCHED)
       .map(async failingNode => {
-        failingNode.cssRule = await fetchSourceRule(driver, failingNode.node);
+        const cssRule = await fetchSourceRule(driver, failingNode.node);
+        // If cssRule is null, then the node was deleted. Don't set `cssRule` in that case.
```

Comment:
how about we just have the try/catch out here so it's a little more self-explanatory and no indentation churn in the function? :)